api/.dockerignore

```diff
@@ -0,0 +1,7 @@
+.idea
+.env
+README.md
+Makefile
+Dockerfile
+.dockerignore
+certs
```

Comment:
```suggestion
certs
.git
.gitignore
docker-compose*
```

Reply:
the docker-compose files in the root /artifacts, not under the /api folder ;)